src/app/containers/StoryPromo/index.jsx

```diff
@@ -11,13 +11,41 @@ import FigureContainer from '../Figure';
 
 import { ServiceContext } from '../../contexts/ServiceContext';
 import deepGet from '../../lib/utilities/deepGet';
+import createSrcset from '../Image/helpers/srcSet';
+
+const pathToArray = path => path.slice(1).split('/');
+
+const getOriginCode = path => {
+  const items = pathToArray(path);
+  const [originCode] = items;
+  if (!originCode) {
+    return null;
+  }
+  return originCode;
+};
+
+const getLocator = path => {
+  const items = pathToArray(path);
+  if (items.length === 0) {
+    return null;
+  }
+  const locator = items.slice(1).join('/');
+  return locator;
+};
 
 const StoryPromoImage = ({ imageValues, lazyLoad }) => {
   if (!imageValues) {
     return null;
   }
-  const ratio = (imageValues.height / imageValues.width) * 100;
-  const src = `https://ichef.bbci.co.uk/news/660${imageValues.path}`;
+
+  const { height, width, path } = imageValues;
+
+  const ratio = (height / width) * 100;
+  const originCode = getOriginCode(path);
+  const locator = getLocator(path);
+  const srcset = createSrcset(originCode, locator, width);
+
+  const src = `https://ichef.bbci.co.uk/news/660${path}`;
```

Comment:
Can you make `660` a variable with the name "Default image width" please :) Bit clearer